src/as-validator.c

```diff
@@ -2089,6 +2089,12 @@ as_validator_check_release (AsValidator *validator, xmlNode *node, AsFormatStyle
 			continue;
 		}
 
+		/* checks if the description is put outside a description tag */
+		if (g_strcmp0 (node_name, "p") == 0 || g_strcmp0 (node_name, "ol") == 0 || g_strcmp0 (node_name, "ul") == 0 || g_strcmp0 (node_name, "li") == 0) {
```

Comment:
```suggestion
		if (as_str_equal0 (node_name, "p") || as_str_equal0 (node_name, "ol") || as_str_equal0 (node_name, "ul") || as_str_equal0 (node_name, "li")) {
```
Just for nicer readability :-)

Reply:
Historical reasons - `as_str_equal0` is a later addition. Likely a Coccinelle script could be written to auto-adjust, but it's not that important.
For new code though, using the helper is nice ;-)